keps/sig-api-machinery/20190612-crd-group-protection.md

```diff
@@ -0,0 +1,139 @@
+---
+title: k8s.io Group Protection
+authors:
+  - "@deads2k"
+owning-sig: sig-api-machinery
+participating-sigs:
+  - sig-api-machinery
+  - sig-architecture
+reviewers:
+  - "@sttts"
+  - "@jpbetz"
+  - "@liggitt"
+approvers:
+  - "@liggitt"
+  - "@sttts"
+editor:
+  name: "@deads2k"
+creation-date: 2019-06-12
+last-updated: 2019-07-03
+status: implementable
```

Comment:
KEPs should be checked in early as `provisional`, iterated and *then* be transitioned to implementable.  By having the initial PR mark the KEP as implementable means that the thing is all-or-nothing and it is more difficult to converge and merge.